examples/regression/4-tutorials/plot_main-tutorial-regression.py

```diff
@@ -682,124 +681,9 @@ def get_1d_data_with_normal_distrib(funct, mu, sigma, n_samples, noise):
 
 
 ##############################################################################
-# 4. Estimating the uncertainty with different sklearn-compatible regressors
```

Comment:
Is it impossible to keep this example directly in the documentation?

Reply:
this notebook has too great an impact on memory and time complexity (tensorflow import, for example, crashes everything)